src/ls/ls.rs

```diff
@@ -196,6 +196,7 @@ fn sort_entries(entries: &mut Vec<PathBuf>, options: &getopts::Matches) {
                     .unwrap_or(std::time::UNIX_EPOCH)
             });
         }
+        entries.reverse(); // Newest first
```

Comment:
Perhaps it makes more sense to change `sort_by_key` above into:
```rust
entries.sort_by(|a, b| {
    get_mod_time(b, options).cmp(get_mod_time(a, options)))
});
```
where `get_mod_time()` is defined as:
```rust
fn get_mod_time(path: &PathBuf, options: &getopts::Matches) -> time::SystemTime {
    get_metadata(k, options)
        .and_then(|md| md.modified())
        .unwrap_or(std::time::UNIX_EPOCH)
}
```

That way we don't have to reverse the vector.  Of course, you would also have to define a similar `get_create_time` method or something to that effect for Unix.